drivers/i2c/i2c_dw.c

```diff
@@ -197,9 +197,8 @@ static inline void i2c_dw_write_byte_non_blocking(const struct device *dev, uint
 static void i2c_dw_slave_read_clear_intr_bits(const struct device *dev);
 #endif
 
-static void i2c_dw_isr(void *arg)
+static void i2c_dw_isr(const struct device *port)
```

Comment:
dev?

Reply:
"port" was kept as this is the name used already for the device for the code (see below: arg was casted to a struct device *port).